front_end/src/app/(main)/questions/[id]/components/forecast_maker/forecast_maker_conditional/forecast_maker_conditional_binary.tsx

```diff
@@ -241,6 +261,31 @@ const ForecastMakerConditionalBinary: FC<Props> = ({
     }
   };
 
+  const handlePredictWithdraw = async () => {
+    setSubmitErrors([]);
+
+    if (!prevYesForecastValue && !prevNoForecastValue) return;
+
+    const response = await withdrawForecasts(postId, [
+      ...(prevYesForecastValue ? [{ question: questionYesId }] : []),
+      ...(prevNoForecastValue ? [{ question: questionNoId }] : []),
+    ]);
+    questionOptions.forEach((q) => {
+      q.isDirty = false;
+    });
```

Comment:
`questionOptions` is react state and it shouldn't be mutated. You should use `useState` for updating it's value. For reference, see how this is done in submission callback:

```
    setQuestionOptions((prev) =>
      prev.map((prevChoice) => ({ ...prevChoice, isDirty: false }))
    );
```